modules/FvwmPager/FvwmPager.c

```diff
@@ -1541,19 +1547,9 @@ void list_config_info(unsigned long *body)
 		m = fxcalloc(1, sizeof(*m));
 
 		m->name = fxstrdup(mname);
-		m->x = x;
-		m->y = y;
-		m->w = w;
-		m->h = h;
-		m->is_current = iscur;
-		m->output = output;
-		m->virtual_scr.MyDisplayWidth = mdw;
-		m->virtual_scr.MyDisplayHeight = mdh;
-		m->virtual_scr.Vx = vx;
-		m->virtual_scr.Vy = vy;
-		m->virtual_scr.VxMax = vxmax;
-		m->virtual_scr.VyMax = vymax;
+        extract_monitor_config(m,tline);
```

Comment:
Same here -- indentation looks odd.  Also:

```suggestion
        extract_monitor_config(m, tline);
```